### ISSUE TITLE
[Bug] Scan active icon color is not white, has poor contrast

### ISSUE THREAD

**Issue by yoasif** [mozilla-mobile/fenix#14861]
## Steps to reproduce

1. Open Fenix
2. Tap address bar
3. Tap Scan button

### Expected behavior

Upon tapping, the scan icon should become white, like the search engine icon becomes. 

### Actual behavior

Upon tapping, the scan icon remains black and has poor contrast for a moment before the screen becomes a viewfinder. 

### Device information

* Android device: Moto G XT1031
* Fenix version: release 80, beta, nightly
